nbconvert/filters/strings.py

```diff
@@ -75,6 +78,21 @@ def html2text(element):
     return text
 
 
+def clean_html(element):
+    if isinstance(element, bytes):
+        element = element.decode()
+    else:
+        element = str(element)
+    return bleach.clean(
+        element,
+        tags=[*bleach.ALLOWED_TAGS, "div", "pre", "code", "span"],
```

Comment:
This `clean_html` function is clearly unsuitable to be called on `image/svg+xml` elements, since it will escape all xml tags.   Unfortunately the `base.html.j2` templates call `clean_html` in this context.  See #1849.

Rather than listing tags that are allowed, would it be possible to list tags that are disallowed?  (Like I suspect `<script>` and `<iframe>` are unwelcome. )

Reply:
Ah, that's a bummer. No tests clearly covered this, so that's a regression. SVG tags are well-known though so they could be added to the allowlist...